### PR TITLE
Send shell done

### PR DESCRIPTION
**Pull request by HectorNyblom**

### CODE REVIEW

**Comment by HectorNyblom** [comment]
Updated consistensy for use of datatype in view contract

File: flaskapp/shell/form.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            clientlist.append((str(client.id), client.name+", Org.nr.: "+str(client.id)))
+        return clientlist
+
+    def getConditions(self):
+        conditions = []
+        for condition in Conditions.query.all():
+            conditions.append((str(condition.id), condition.name))
+        return conditions
+    
+    def getFileList(self, client_id):
```
Comment (chopan-7) [inline flaskapp/shell/form.py:26]: remove getFileList() before merging to master

File: flaskapp/shell/form.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        }
+
+        try:
+            db.session.autocommit = False
+            db.session.add(new_shell)
+            db.session.commit()
+            with open(app.config['SHELL_FOLDER']+str(new_shellID)+app.config['SHELL_FILEEXT'], 'w') as outfile:
+                json.dump(json_contract, outfile, indent=4)
+            
+            #send_contract(new_contractID, clientID)
```
Comment (chopan-7) [inline flaskapp/shell/form.py:75]: remove send_contract() call before merging to master

**Review by chopan-7 — COMMENTED** [review]
Beautifully written code! <3!

**Pull request merged**